internal/testutils/integrationHelpers.go

```diff
@@ -71,3 +79,10 @@ func Cleanup(t *testing.T, dbConn *gorm.DB) {
 		t.Fatalf("error cleaning db after test run: %v", err)
 	}
 }
+
+// Cleanup intended to be called in between subtests
+func BeforeIntegrationSubTestCleanup(t *testing.T, dbConn *gorm.DB) {
```

Comment:
Can we just call this cleanup? it isn't necessarily specific to subtests

Reply:
probably, this was going to be specific to subtests, but then I just ended up using Cleanup